Makefile

```diff
@@ -19,3 +19,15 @@ promlinter-build:
 .PHONY: promlinter-push
 promlinter-push:
 	${CONTAINER_RUNTIME} push ${IMG}
+
+.PHONY: monitoringlocationlinter-unit-test
+monitoringlocationlinter-unit-test:
+	cd test/monitoringlocationlinter && go test ./...
+
+.PHONY: monitoringlocationlinter-build
+monitoringlocationlinter-build:
+	cd test/monitoringlocationlinter && go build ./cmd/monitoringlocationlinter
+
+.PHONY: monitoringlocationlinter-test
+monitoringlocationlinter-test: monitoringlocationlinter-build
+	cd test/monitoringlocationlinter && ./tests/e2e.sh
```

Comment:
not sure is also correct, but what if we created a Makefile in `test/monitoringlocationlinter` instead

Reply:
We can, but not sure it will be convenient to have to `cd test/monitoringlocationlinter/ && make monitoringlocationlinter-unit-test` in order to run those targets. What is the benefit of having another Makefile?

Reply:
```
.PHONY: monitoringlocationlinter-unit-test
monitoringlocationlinter-unit-test:
  ...
```

the benefit is not having this kind of definitions


and should read:
`cd test/monitoringlocationlinter/ && make unit-test`

Reply:
Didn't create another Makefile, but now target names are shorter.